train_management/tests/helpers.py

```diff
@@ -2,6 +2,7 @@
 from train_management.models import Train
 from train_management.models import Vehicle
 from train_management.models import TrainConfiguration
+from train_management.models import Function
```

Comment:
I think `Function` is a confusing naming...

Reply:
renamed in next pr